src/gateway/shard.rs

```diff
@@ -12,22 +12,22 @@ use websocket::stream::sync::AsTcpStream;
 use websocket::sync::client::{Client, ClientBuilder};
 use websocket::sync::stream::{TcpStream, TlsStream};
 use websocket::WebSocketError;
-use constants::{self, close_codes, OpCode};
-use internal::prelude::*;
-use internal::ws_impl::SenderExt;
-use model::event::{Event, GatewayEvent};
-use model::{Game, GuildId, OnlineStatus};
+use ::constants::{self, OpCode, close_codes};
+use ::internal::prelude::*;
+use ::internal::ws_impl::SenderExt;
+use ::model::event::{Event, GatewayEvent};
+use ::model::{Game, GuildId, OnlineStatus};
 
-#[cfg(feature = "voice")]
+#[cfg(feature="voice")]
```

Comment:
The same can be said to these formatting changes too.
Although to make it a bit painless, use rustfmt.